spec/GraphQLOverHTTP.md

```diff
@@ -166,6 +166,9 @@ The GraphQL specification allows for many
 Servers and clients MUST support JSON and MAY support other, additional
 serialization formats.
 
+Note: If you allow other media types, particularly on requests, please ensure
+that you are aware of the security consequences of doing so.
```

Comment:
```suggestion
Note: Allowing other media types, particularly on requests, can be insecure.
```

Reply:
It's more like "can be insecure if you don't take the necessary precautions/actions".